Makefile

```diff
@@ -1,5 +1,8 @@
 artifact_name       := test-data-generator
-version             := "unversioned"
+version             := unversioned
+docker_registry     := ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com
+docker_repository   := ${artifact_name}
+exposed_port        := 10000
```

Comment:
If we're using `local_env` I'd specify this via `${TEST_DATA_GENERATOR_PORT}`

Reply:
Yep, that's a good shout. Updated the exposed_port var to assign from TEST_DATA_GENERATOR_PORT